Moq.AutoMock.Tests/Util/With3Parameters.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace Moq.AutoMock.Tests.Util
 {
 #pragma warning disable CA1801, CA1812  //is an internal class that is apparently never instantiated
-    internal class With3Parameters
+    public class With3Parameters
```

Comment:
There is a bit of a disjoint here. If this should be `public` then we should remove the CA1812 above and update the comment, or this could go back to `internal`